cdap-ui/app/cdap/components/OpsDashboard/store/ActionCreator.js

```diff
@@ -26,15 +26,17 @@ export function enableLoading() {
   });
 }
 
-export function getData() {
+export function getData(start, duration = 1440) {
   enableLoading();
 
   let state = DashboardStore.getState().dashboard;
 
-  let start = moment().subtract(24, 'h').format('x'),
-      duration = 1440;
+  if (!start) {
+    start = moment().subtract(24, 'h').format('x');
+    start = parseInt(start, 10);
 
-  start = parseInt(start, 10);
+    duration = 1440;
```

Comment:
nit: unnecessary, since you already set default `duration = 1440` in the params?